db/helpers/seed_helper.rb

```diff
@@ -0,0 +1,29 @@
+class SeedHelper
+  def objectify(model)
+    return model.titleize.constantize unless model.include? "_"
```

Comment:
I just tested it out now. It works well. But I could only test with strings that have no `_` as I don't know any of our models that contain that.